go.mod

```diff
@@ -1,4 +1,4 @@
-module example.com/terraform-provider-logdna
+module github.com/logdna/terraform-provider-logdna
```

Comment:
just noting that we can't release without this change being merged, if this PR doesn't land we will need a separate PR for the changes here and in `main.go` - not sure what the priority is for these bodyTemplate changes w/r/t release.

Reply:
This PR is indeed a release blocker, are you saying that we can't release without this PR landing?

Reply:
i'll make a separate pr out of this if necessary. you're right that it doesn't have to be decoded into a map[string]interface{}

Reply:
@michaelshi-logdna @mdeltito saying we can't release without fixing the module example.com issue